File: PHP/Math.php
<?php
// Operações matemáticas em PHP

$a = 15;
$b = 4;

// Operações básicas
echo "Soma: " . ($a + $b) . "<br>";
echo "Subtração: " . ($a - $b) . "<br>";
echo "Multiplicação: " . ($a * $b) . "<br>";
echo "Divisão: " . ($a / $b) . "<br>";
echo "Módulo (resto da divisão): " . ($a % $b) . "<br>";

// Potência
echo "Potência: " . pow($a, $b) . "<br>";

// Raiz quadrada
echo "Raiz quadrada de $a: " . sqrt($a) . "<br>";

// Valor absoluto
echo "Valor absoluto de -$a: " . abs(-$a) . "<br>";

// Arredondamentos
echo "Arredondar 3.6: " . round(3.6) . "<br>";
echo "Arredondar para cima 3.2: " . ceil(3.2) . "<br>";
echo "Arredondar para baixo 3.8: " . floor(3.8) . "<br>";

// Número aleatório entre 1 e 10
echo "Número aleatório entre 1 e 10: " . rand(1, 10) . "<br>";
?>

File: C++/Básico/Math.cpp
#include <iostream>
#include <cmath> // Necessário para funções matemáticas

int main() {
    double x = 9.0;
    double y = 2.0;

    // Funções matemáticas básicas
    double raiz = sqrt(x);         // Raiz quadrada
    double potencia = pow(x, y);   // Potência (x elevado a y)
    double absoluto = fabs(-7.5);  // Valor absoluto
    double seno = sin(x);          // Seno
    double cosseno = cos(x);       // Cosseno
    double logaritmo = log(x);     // Logaritmo natural

    std::cout << "Raiz quadrada de " << x << ": " << raiz << std::endl;
    std::cout << x << " elevado a " << y << ": " << potencia << std::endl;
    std::cout << "Valor absoluto de -7.5: " << absoluto << std::endl;
    std::cout << "Seno de " << x << ": " << seno << std::endl;
    std::cout << "Cosseno de " << x << ": " << cosseno << std::endl;
    std::cout << "Logaritmo natural de " << x << ": " << logaritmo << std::endl;

    return 0;
}